docs/cloud-native-security/cspm-get-started-aws.asciidoc

```diff
@@ -30,6 +30,21 @@ This page explains how to get started monitoring the security posture of your cl
 
 You can set up CSPM for AWS either by enrolling a single cloud account, or by enrolling an organization containing multiple accounts. Either way, first you will add the CSPM integration, then enable cloud account access.
 
+[discrete]
+[[cspm-aws-agentless]]
+== Agentless deployment option
```

Comment:
I would suggest adding a header for agentless deployment option and capture the methods supported under that section. Also, as per the beta guide shared, you could also add some description of the agentless method and when user would choose this.

Reply:
I don't want to explain the entire authentication methods twice on the same page (once for agentless and once for agent-based), but I added more detail about the steps other than authentication to this section, and link to the authentication methods.